pymc_marketing/clv/distributions.py

```diff
@@ -2,14 +2,11 @@
 import pymc as pm
 import pytensor.tensor as pt
 from pymc.distributions.continuous import PositiveContinuous
-from pymc.distributions.dist_math import check_parameters
+from pymc.distributions.dist_math import betaln, check_parameters
+from pytensor import scan
 from pytensor.tensor.random.op import RandomVariable
 
-__all__ = [
-    "ContContract",
-    "ContNonContract",
-    "ParetoNBD",
-]
+__all__ = ["ContContract", "ContNonContract", "ParetoNBD", "BetaGeoBetaBinom"]
```

Comment:
If you add a comma after the last item black will let it stay one item per line, which I think is better for git diffs